scripts/helm_docs.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Also, please rename this file to `git-hook/helm-docs`, and update the `.pre-commit-hooks.yaml` file accordingly. I think it's better to make clearer that this script is only intended for the git hooks.

Reply:
Yup, I will tomorrow, don't have laptop with me tonight.

Reply:
Done.